index.css

```diff
@@ -504,3 +567,370 @@ textarea {
 .formato {
   height: 400px;
 }
+
+@media (min-width: 768px) {
+  nav {
+    display: none;
+  }
+
+  .active {
+    color: #fff;
+    width: 92px;
+    height: 20px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 20px;
+    text-decoration: none;
+    margin: 28px 0 0 120px;
+  }
+
+  .topnav {
+    display: flex;
+    justify-content: space-between;
+    color: #fff;
+    background: #3e3c3c url(Desktop/mail.png) 91% 62% no-repeat;
+    height: 92px;
+  }
+
+  .topnav #myLinks {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    color: #f2f2f2;
+    padding: 14px 50px 0 0;
+    margin-right: 110px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .roger {
+    height: 870px;
+    display: flex;
+    padding: 0 0 0 37px;
+    background: #1c1a19 url(Desktop/muñeco2.png) 100% 125% no-repeat;
+  }
+
+  .pad {
+    width: 566px;
+    height: 160px;
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-size: 50px;
+    font-weight: 400;
+    line-height: 82px;
+    margin-top: 130px;
+    margin-left: 80px;
+    color: #f55800;
+  }
+
+  .supporttest {
+    color: #fff;
+    width: 566px;
+    height: 24px;
+    margin: 20px 0 0 90px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+  }
+
+  .developer {
+    margin: 30px 0 0 0;
+  }
+
+  .theme1 {
+    color: #f4f5f7;
+    width: 440px;
+    height: 123px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  .palet {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: center;
+    list-style: none;
+  }
+
+  .palet a {
+    position: relative;
+    bottom: 270px;
+  }
+
+  .palet li {
+    height: 45px;
+  }
+
+  .myresent {
+    display: flex;
+    margin-left: 120px;
+    background-image: url(../Desktop/line3.png);
```

Comment:
- [ ] Same here remove one `.` from the start of the URL path.

Reply:
ok

Reply:
thank you for your CR